editor/editor_file_system.cpp

```diff
@@ -3113,10 +3207,9 @@ Error EditorFileSystem::make_dir_recursive(const String &p_path, const String &p
 	const String path = da->get_current_dir();
 	EditorFileSystemDirectory *parent = get_filesystem_path(path);
 	ERR_FAIL_NULL_V(parent, ERR_FILE_NOT_FOUND);
+	folders_to_sort.insert(parent->get_instance_id());
```

Comment:
If you remove `folders_to_sort`, maybe it's better to revert back to add the directory directly at the right position in this method.

Reply:
I added sorting, because I had problems with adding at the right position.